packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -60,24 +59,24 @@ export class TileDoctype extends Doctype {
      * @param context - Ceramic context
      * @param opts - Initialization options
      */
-    static async makeGenesis(params: Record<string, any>, context?: Context, opts: DocOpts = {}): Promise<Record<string, any>> {
+    static async makeGenesis(params: DocParams, context?: Context, opts: DocOpts = {}): Promise<Record<string, any>> {
         if (!context.user) {
             throw new Error('No user authenticated')
         }
 
-        let { owners } = params
-        const { content } = params
-
-        if (!owners) {
-            owners = [context.user.DID]
-        }
-
         let unique: string
         if (opts.isUnique) {
             unique = base64Encode(randomBytes(12))
         }
 
-        const record = { doctype: DOCTYPE, owners, content, unique }
+        const metadata = params.metadata? params.metadata : {}
+        const { owners } = metadata
+        if (!owners) {
+            metadata.owners = [context.user.DID]
+        }
+
+        const { content } = params
+        const record = { doctype: DOCTYPE, data: content, metadata, unique }
```

Comment:
Same thing here, lets use the `header` property in the *records*, and use the `metadata` property in *DocState*.

Reply:
👍